sys/include/net/sock/dtls.h

```diff
@@ -507,6 +507,12 @@
  * registered credential in the Sock's credential list, that matches the needed type. The first
  * one that matches is used.
  *
+ * ### Public key verification
+ *
+ * By enabling the pseudomodule `sock_dtls_verify_public_key` the DTLS sock will verify the
+ * public key of the remote peer. When enabled, the DTLS sock will only accept a connection if
+ * the provided public key is in the list of public keys assigned to the specified sock.
```

Comment:
This is only true for ECDSA, right? 
AFAIK with the default `CREDMAN_TYPE_PSK` there are no public/private keys, only symmetric encryption.

Reply:
I don't think that's obvious (it certainly was not for me) that `CREDMAN_TYPE_PSK` is symmetric only whereas `CREDMAN_TYPE_ECDSA` will do a Diffie-Hellman handshake (and use symmetric crypto afterwards)